src/main/groovy/com/blackbaud/templates/project/PerformanceTestsProject.groovy

```diff
@@ -15,6 +15,10 @@ class PerformanceTestsProject {
 
 # Performance test results
 results/
+""")
+        File gradleSettingsFile = basicProject.getProjectFileOrFail("settings.gradle")
```

Comment:
settings.gradle is not guaranteed to exist

Reply:
Fixed